gammapy/maps/axes.py

```diff
@@ -3030,6 +3030,25 @@ def to_gti(self):
             self.edges_min, self.edges_max, reference_time=self.reference_time
         )
 
+    def to_table(self, format="gadf"):
+        """Create table.
+
+        Parameters
+        ----------
+        format : {"gadf"}
+            Format specification. Default is "gadf".
```

Comment:
Is there a specification for such a table in GADF? Usually time quantities are stored with a reference value stored as MJDREFF in the header and start and stops are just time like quantities.

Reply:
You are right, the specification would be to give the time as a quantity, like doing `to_gti().table`. That is probably the best solution.